docs/components/connectors/out-of-the-box-connectors/embeddings-vector-db.md

```diff
@@ -55,6 +55,11 @@ To perform this operation, enter the following:
 As a result of this operation, you will get an array of created embedding chunk IDs,
 for example `["d599ec62-fe51-4a91-bbf0-26e1241f9079", "a1fad021-5148-42b4-aa02-7de9d590e69c"]`.
 
+:::important
+Every time you embed a document, the connector creates a new set of chunks and stores them in the vector database.
+To update the existing document, first delete the previous chunks.
+:::
```

Comment:
Maybe to `update an existing document`?

Can we further clarify how we would delete these chunks? Would this be something which needs to be done on the storage side manually? Which attributes can be used to identify which chunks need to be deleted?

Reply:
Yes, the removal has to be done manually on the storage side. Currently the only way to know which chunks need to be deleted is to record the chunk IDs that are returned at the end of embedding action. If the embedding source is a Camunda document we could also use the document file name that is stored as a metadata in the stored chunks.
Should we add this clarification to the doc?